compair/static/modules/login/login-partial.html

```diff
@@ -1,22 +1,20 @@
 <div class="login-screen">
     <div ng-show="!showCreateUserForm">
-        <h2 class="text-center">Welcome to ComPAIR</h2>
+        <h1 class="text-center">Welcome to ComPAIR</h1>
         <div class="row">
             <form name="loginForm" class="form col-sm-6" ng-submit="submit()" novalidate>
                 <fieldset>
                     <div ng-if="AuthTypesEnabled.cas">
-                        <h3>Log in with your CWL here:</h3>
-                        <a id="cwl_login" href="/api/cas/login">
-                            <img class="center-block" src="https://www.auth.cwl.ubc.ca/CWL_login_button.gif" width="76" height="25" alt="CWL Login" border="0" valign="middle">
```

Comment:
added CWL image back for now https://github.com/ubc/compair/pull/558/commits/1c1f7c8d1030e35b9598d680ce270b89c3461064